data/jibauni/jibauni 0 kanlanmo transport.txt

```diff
@@ -0,0 +1,670 @@
+			choice
+				`	"Why not? There's nothing else to do."`
+					accept
+				`	"In the sake of establishing a good relationship, sure!"`
+					accept
+				`	"Of course, but unfortunately I don't have space right now. I can come back later though."`
+			`	"Okay," says Kanlanmo. "Chances are, I still won't have found transport, so I will be waiting here next time you come."`
+				decline
+			label notseen
+			`	"Okay. In that case, I have another request for you. Can you take me to <planet> in the <system> system? I will mark it on your map, and explain more once we get there. Oh yes, and my name is Kanlanmo. Kanlanmo of the Jibauni."`
+			choice
+				`	"Of course! You can get on the <ship> right away!"`
+					accept
+				`	"Why?"`
+					goto whychoice
+				`	"I'm sorry, I can't take you now. Later maybe though."`
+					decline
+			label wormholeentry
+			`	"Oh, we have seen it," says the alien. "But it does not let our ships though. The reason for this is unclear. but to matters of more importance. We have seen an alien ship in pink markings pass through. Have you seen it, or one like it?"`
+				goto pinkship
+			label alphas
+			`	You begin the story, telling everything you know about how the Alphas tried to enslave human space.`
+			`	At the end, he says, "Wow. That is a long story. And I take it you are not on their side?"`
+			choice
+				`	"No, of course not."`
+					goto alphasoffer
+				`	"Yes, I have been friendly with them for years."`
+					goto friendlyalpha
+			label alphasoffer
+			`	"Good. Well, that just leaves one more matter to be wrapped up. Can you take me to <planet> in the <system> system? Kanlanmo of the Jibauni will be grateful."`
+			choice
+				`	"Of course!"`
+					accept
+				`	"Why?"`
+					goto whychoice
+				`	"Not yet, I have something I need to do first. But I'll be back shortly!"`
+					decline
+			label rude
+			`	The alien announces, though to what you cannot see, "The stranger has been ignominiously impolite." It turns to you. "What do you say, 'weirdo'? Will you apologize? Or not, and see what happens?"`
+			choice
+				`	"I apologize, and did not mean to insult you."`
+					goto rudeentry
+				`	"Why apologize to a bunch of dinky T-Rex's?"`
+					goto deathexplode
+			label deathexplode
+			`	The moment the last word is out of your mouth, the dinosaur lowers its hand. Suddenly, millions of miniature projectiles come hurtling through the air, shredding the <ship> and everything in it. It is sad that your life had to end this way.`
+				die
+			label deathshot
+			`	The moment your hand touches your sidearm, a whining noise is heard. A huge missile comes down with a shriek, followed by another and another. <ship> is obliterated in a moment, leaving you in a pile of smoldering ash. You are taken to jail and imprisoned for life.`
```

Comment:
Just reminding the player that First Last isn't in the ship, and slight tweak as First Last would certainly be dead from the explosion if it left _them_ surrounded by a pile of ash.
```suggestion
			`	The moment your hand touches your sidearm, a whining noise is heard. A huge missile comes down with a shriek, followed by another and another. The <ship> behind you is obliterated in a moment, leaving it a pile of smoldering ash. You are taken to jail and imprisoned for life.`
```